datascience_example.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>It seems that there are only comments, and that they are only for blocks.</p><p> </p><p>There are no suggestions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/guaraqe/reviewnb-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
And these comments appear on GitHub too, that's nice. But they are less useful.

Reply:
<p>True. But with these limitations, I'm not sure whether it's eventually a really useful tool for us. Suggestions would be really, really nice and comments on something other than whole cells, too. A halfway solution for that latter would be to try to make as many cells as possible in a notebook, but as each cell makes a new paragraph with space between the previous and next ones (I don't know the term for that), that gives us a lower limit on "comment resolution".</p>